contracts/src/v0.8/functions/dev/1_0_0/HasRouter.sol

```diff
@@ -42,11 +44,19 @@ abstract contract Routable is ITypeAndVersion, IConfigurable {
   function _updateConfig(bytes memory config) internal virtual;
 
   /**
-   * @inheritdoc IConfigurable
+   * @dev Internal function that can be re-used in the constructor
    */
-  function updateConfig(bytes memory config) external override onlyRouter {
+  function _handleConfigUpdate(bytes memory config) private {
```

Comment:
Not sure that I love the addition of this helper function, but I see the notes [here](https://www.notion.so/Steve-s-Functions-Solidity-Review-Notes-64447db7e6a64ad9a5f0ce503186eef6?d=437f97af79e04d4dafd1198bfe9f0120&pvs=4#b5976ba516244674849b393626379002) and will refrain from blocking the review because of it.

Reply:
Do prefer how it was before with the duplicated code in the constructor? (I don't have a strong opinon here)

Reply:
I also don't love the third config setting function with a fairly confusing name.

Reply:
Rolling it back unless someone has a suggestion of how to fulfill Steve's point